Include/stories.php

```diff
@@ -0,0 +1,44 @@
+<?php
+    function getAllStories(){
+        $stories = dbQuery("
+            SELECT * FROM `stories`
+        ")->fetchAll(); 
+
+        return $stories; 
+    }
+
+    function getStory($storyId){
+        $AllStories = getAllStories(); 
+        return $AllStories[$storyId]; 
+    }
+
+    function getChapter($chapterId){
+        $chapter = dbQuery("
+        SELECT * 
+        FROM `chapters`
+        WHERE `chapterId` = $chapterId
+        ")->fetch(); 
+        return $chapter; 
+    }
+
+    function getFirstChapter($storyId){
+
+        $chapter = dbQuery(" 
+         SELECT *
+         FROM chapters
+         WHERE StoryId = $storyId 
```

Comment:
Same comment here re: escaping variables instead of using them directly in your queries in order to avoid vulnerabilities like sql injections